firedrake/mesh.py

```diff
@@ -77,7 +77,8 @@ def set(self):
             else:
                 base = self.mesh._base_mesh.exterior_facets.set
             return op2.ExtrudedSet(base, layers=self.mesh.layers)
-        return op2.Set(size, "%s_%s_facets" % (self.mesh.name, self.kind),
+        return op2.Set(size, "%s%sFacets" % (self.mesh.name,
+                                             self.kind.capitalize()[:3]),
```

Comment:
Agreed. I also re-instated the "ParLoop" part of the tag in the PyOP2 PR, so that we now get the following output from `-log_view`:
```
ParLoopExecute       240 ...
ParLoopCells         240 ...
ParLoopHaloEnd       240 ...
ParLoopReductionBegin     240 ...
ParLoopReductionEnd     240 ...
ParLoopIntFacets     240 ...
ParLoopExtFacets     240 ...
```